pandas/core/frame.py

```diff
@@ -3565,7 +3575,6 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None,
              A    B   C  D
         0  NaN  2.0 NaN  0
         1  3.0  4.0 NaN  1
```

Comment:
It would be nice to add some more examples. Starting by creating a simple dataframe, using `dropna` without parameters, and then explain and show examples using the parameters. In the documentation for the sprint you should have examples.